examples/with-rspack/README.md

```diff
@@ -0,0 +1,31 @@
+# With Polyfills
+
+Next.js supports IE11 and all modern browsers (Edge, Firefox, Chrome, Safari, Opera, et al) with no required configuration. It also adds [some polyfills](https://nextjs.org/docs/architecture/supported-browsers#polyfills) by default.
```

Comment:
Guessing this is copied README but looks like we might need to update this list in all our examples cc @leerob @feedthejim 

Maybe we just link here https://nextjs.org/docs/architecture/supported-browsers

Reply:
^ Agree we should link to that docs page and ensure it's still accurate.

Reply:
Resolving this in https://github.com/vercel/next.js/pull/77211